ui/src/routing.js

```diff
@@ -57,8 +55,9 @@ class Routing extends React.Component {
                 this.signInUser(token);
                 return;
             }
-            document.cookie = 'application=; expires=' + new Date(0).toUTCString();
-            window.location.reload();
+            User.signOut().finally(() => {
+                window.location.reload();
```

Comment:
Will this work properly for login protected pages? I wouldn't want to see an "Access denied" error when I log out :)

Reply:
I think it does because every page is login protected. We will be redirected only to The login page when a user signs out, even if its due to session timeout or a explicit logout. The ui does not have a forbidden page in use yet. Im not entirely sure when to use it. We dont have any views that are restricted to specific users.

Reply:
Alright, good enough until we do then :)